docs/src/main/sphinx/release/release-372.md

```diff
@@ -0,0 +1,89 @@
+# Release 372 (23 Feb 2022)
+
+## General
+
+* Remove support for reserved memory pool. Configuration property
+  `experimental.reserved-pool-disabled` can no longer be used. ({issue}`6677`)
+* Ensure memory is released completely after query completion. ({issue}` 11030`)
+
+## Base-JDBC connector library
+
+* Fix spurious query failures when metadata cache is not enabled, and data
+  access depends on the session state. ({issue}`11068`)
```

Comment:
what does the second clause mean?

Reply:
I think this is the fix in the base class that results in 

* Fix spurious query failures when metadata cache is not enabled, and extra
  credentials with `user-credential-name` or `password-credential-name` are used
  to access data. ({issue}`11068`)

so it tries to be more generic in wording. I am not sure this make sense to add at all to be honest. Also not sure how to reword. Any input @findepi ?

Reply:
@dain Connectors based on base-jdbc can have a feature where for-example some session property causes the types to be mapped in a different way, or to maybe to switch the active "catalog" for systems which expose both schemas and catalogs.

In such connectors it's possible that a failure in one session can be propagated to other session (Guava cache behaviour).

We have documented user-facing changes in this library in the past for quite a lot of versions already and pretty consistently. IMO we should document this since it is a user facing bug fix (for any connectors build on base-jdbc).

Reply:
Note that theoretically this can happen in any connector (e.g. Hive) which uses the Guava caches and tries to disable the cache instead of removing it entirely since accesses to the Guava cache are synchronized even if the cache is disabled.

Reply:
So is the existing release notes wording ok?